cmd/manager/main.go

```diff
@@ -225,3 +239,50 @@ func checkIfRequiredFilesExist(requiredFiles []string) error {
 	}
 	return nil
 }
+
+// clusterConfigChecks checks for required configuration in a cluster before starting the controller.
+// If the cluster doesn`t meet the configuration, it returns an error.
+func clusterConfigChecks(cfg *rest.Config) error {
+	// Get the OpenShift API config client.
+	clienset, err := configclient.NewForConfig(cfg)
+	if err != nil {
+		return errors.Wrap(err, "error creating the OpenShift API config client")
+	}
+
+	// Get the OpenShift operator client
+	operatorClient, err := operatorv1.NewForConfig(cfg)
+	if err != nil {
+		return errors.Wrap(err, "error creating the OpenShift API operator client")
+	}
+	// Check for valid K8s version. Currently any version >= 1.18 is supported.
```

Comment:
We support any 1.18.x versions but we will not support 1.19.x in the 4.5 operator.